R/reexports.R

```diff
@@ -0,0 +1,8 @@
+#' @export
+dplyr::vars
```

Comment:
I just noticed that `vars()` is superseded. Thus I think for admiral 1.0.0 we need to revise using `vars()` in admiral.